src/App.tsx

```diff
@@ -1,61 +1,55 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { ToDo } from './types/ToDo';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { ToDoForm } from './components/ToDoForm/ToDoForm';
+
+function getUserById(userId: number) : User | null {
+  return usersFromServer.find((user) => user.id === userId) || null;
+}
+
+function getFormattedTodos() {
+  return todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }));
+}
+
+const getNewToDoId = (todos: ToDo[]): number => {
+  const maxId: number = Math.max(...todos.map((todo) => todo.id));
+
+  return maxId + 1;
+};
 
 export const App = () => {
+  const [todos, setTodos] = useState<ToDo[]>(getFormattedTodos());
+
+  const addToDo = (userId: number, newTitle: string) => {
+    const user = getUserById(userId);
+    const todo: ToDo = {
+      id: getNewToDoId(todos),
+      title: newTitle,
+      completed: false,
+      userId,
+      user,
+    };
+
+    if (user !== null) {
+      setTodos((currentToDos) => [...currentToDos, todo]);
+    }
```

Comment:
```suggestion
    if (user) {
      setTodos((currentToDos) => [...currentToDos, todo]);
    }
```
You don't need to compare with `null`, just check if exists